packages/frameworks/react/src/tree-view/tree-view-root.tsx

```diff
@@ -19,7 +19,7 @@ export interface TreeViewRootProps
   > {}
 
 export const TreeViewRoot = forwardRef<HTMLDivElement, TreeViewRootProps>((props, ref) => {
-  const [useTreeViewProps, { children, ...divProps }] = createSplitProps<UseTreeViewProps>()(
+  const [useTreeViewProps, { children, ...localProps }] = createSplitProps<UseTreeViewProps>()(
```

Comment:
We should decide on naming this. There are many inconsistent `divProps`, `spanProps` and `localProps` etc throughout our components.

Reply:
@Omikorin I would go with local props since it is interchangeable